### PR TITLE
added MapAssert#containsOnly

### PR DESCRIPTION
**Pull request by alex-dukhno**
#56 `containsOnly` part
@mchmielarz @pivovarit could you, please, take a look
Thanks

### CODE REVIEW

File: src/main/java/org/assertj/vavr/internal/Maps.java [hunk 1]
```diff
@@ -63,15 +65,7 @@ public static Maps instance() {
...
-        final Set<Tuple2<? extends K, ? extends V>> notFound = Array
-                .of(entries)
-                .foldLeft(HashSet.empty(), (set, tuple) -> {
-                    if (actual.contains(tuple)) {
-                        return set;
-                    } else {
-                        return set.add(tuple);
-                    }
-                });
+        final Set<Tuple2<K, V>> notFound = Array.of(entries).filter(notContainFrom(actual)).toSet();
```
Comment (mchmielarz) [inline src/main/java/org/assertj/vavr/internal/Maps.java]: 👍

File: src/main/java/org/assertj/vavr/internal/Maps.java [hunk 2]
```diff
@@ -96,15 +90,7 @@ public static Maps instance() {
...
-        final Set<Tuple2<? extends K, ? extends V>> found = Array
-                .of(entries)
-                .foldLeft(HashSet.empty(), (set, tuple) -> {
-                    if (actual.contains(tuple)) {
-                        return set.add(tuple);
-                    } else {
-                        return set;
-                    }
-                });
+        final Set<Tuple2<K, V>> found = Array.of(entries).filter(actual::contains).toSet();
```
Comment (mchmielarz) [inline src/main/java/org/assertj/vavr/internal/Maps.java]: 👍

File: src/test/java/org/assertj/vavr/api/MapAssert_containsOnly_Test.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import org.opentest4j.AssertionFailedError;
+
+import io.vavr.collection.HashMap;
+import io.vavr.collection.List;
+import io.vavr.collection.Map;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.vavr.api.VavrAssertions.assertThat;
+
+class MapAssert_containsOnly_Test {
```
[the remark this answers is not in the document]
Comment (alex-dukhno) [reply src/test/java/org/assertj/vavr/api/MapAssert_containsOnly_Test.java]: thanks for noticing! added

File: src/test/java/org/assertj/vavr/api/MapAssert_containsOnly_Test.java [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  @Test
+  void should_pass_if_Map_contains_only_given_entry() {
+    final Map<String, String> actual = HashMap.of("key", "value");
+
+    assertThat(actual).containsOnly(List.of(Map.entry("key", "value")));
+  }
+
+  @Test
+  void should_shrug() {
```
[the remark this answers is not in the document]
Comment (alex-dukhno) [reply src/test/java/org/assertj/vavr/api/MapAssert_containsOnly_Test.java]: seems like I tested some use case ... and forgot to remove 😄

**Pull request merged**